<launch>
  <node name="detect_grasps" pkg="gpd_ros" type="detect_grasps" output="screen">

    <!-- What type of point cloud is used and what ROS topic it comes from -->
    <param name="cloud_type"  value="0" /> <!-- 0: PointCloud2, 1: CloudIndexed, 2: CloudSamples -->
    <param name="cloud_topic" value="/camera/depth/color/points" />
    <param name="rviz_topic"  value="plot_grasps" />

    <!-- Action for service-driven, if empty using sensor-driven -->
    <param name="action_name"  value="sample_grasp" />
  
    <param name="robot_base"  value="base_link" />
    <param name="camera_optical"  value="camera_depth_optical_frame" />

    <!-- Filepath to the configuration file for GPD -->
    <!-- <param name="config_file" value="/home/ur5/projects/gpd/cfg/ros_eigen_params.cfg" /> -->
    <param name="config_file" value="$(find gpd_ros)/config/gpd_config.yaml" />

    <param name="filtering_score" value="true" />
    <param name="score_threshold" value="200." />

  </node>
</launch>
